serve/mlc_serve/engine/sampling_params.py

```diff
@@ -9,8 +9,8 @@
 from functools import cached_property
 from typing import Dict, Optional
 
-
 _SAMPLING_EPS = 1e-5
+TOP_LOGPROBS_NUMBER = 5
```

Comment:
Need a better name

Reply:
What about "LOGPROB_TOP_K_MAX"? Any suggestion would be welcome.

Reply:
sounds good